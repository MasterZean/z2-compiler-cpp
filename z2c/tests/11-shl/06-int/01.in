70 << true
70 << false
70 << 4s8
70 << 4u8
70 << 4s16
70 << 4u16
70 << 4
70 << 4u
70 << 4l
70 << 4ul
70 << 4.0f
70 << 4.0
70 << '4'
70 << 4p
