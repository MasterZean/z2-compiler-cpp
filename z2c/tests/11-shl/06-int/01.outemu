Int{1120}
Int{1120}
Int{1120}
Int{1120}
Int{1120}
Int{1120}
Int{1120}
Int{1120}
Int{1120}
