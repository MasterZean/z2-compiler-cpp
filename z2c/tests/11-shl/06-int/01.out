70 << 4;
70 << 4u;
70 << 4;
70 << 4u;
70 << 4;
70 << 4u;
70 << 4ll;
70 << 4ull;
70 << 4u;
DummyClass(1, 4): error:
	can't apply operator '<<' on types: 
		'Int' and 
		'Bool'
DummyClass(2, 4): error:
	can't apply operator '<<' on types: 
		'Int' and 
		'Bool'
DummyClass(11, 4): error:
	can't apply operator '<<' on types: 
		'Int' and 
		'Float'
DummyClass(12, 4): error:
	can't apply operator '<<' on types: 
		'Int' and 
		'Double'
DummyClass(13, 4): error:
	can't apply operator '<<' on types: 
		'Int' and 
		'Char'
